package.json

```diff
@@ -66,17 +66,31 @@
   },
   "scripts": {
     "autolint": "npm run lintfix && npm run csslintfix",
+    "build:components": "cd frontend/components && npm run build",
+    "ci:all": "npm ci && cd frontend/components && npm ci",
+    "ciwin:all": "npm ci && npm run install:components",
     "csslint": "./node_modules/.bin/stylelint **/*.css",
     "csslintfix": "./node_modules/.bin/stylelint **/*.css --fix",
+    "dev:components": "cd frontend/components && npm run dev",
+    "install:all": "npm install && npm run install:components",
+    "install:components": "cd frontend/components && npm install",
     "jest": "jest",
-    "lint": "./node_modules/.bin/eslint .",
-    "lintfix": "./node_modules/.bin/eslint . --fix",
+    "lint": "npm run lintfix:markbind && npm run lint:components",
+    "lint:components": "cd frontend/components && npm run lint",
+    "lint:markbind": "eslint .",
+    "lintfix": "npm run lintfix:markbind && npm run lintfix:components",
+    "lintfix:components": "cd frontend/components && npm run lintfix",
+    "lintfix:markbind": "eslint . --fix",
     "pretest": "npm run lint && npm run csslint",
     "pretestwin": "npm run lint && npm run csslint",
-    "test": "jest && cd test/functional && ./test.sh",
-    "testwin": "jest && cd test/functional && test.bat",
-    "updatetest": "cd test/functional && ./update.sh",
-    "updatetestwin": "cd test/functional && update.bat"
+    "test": "jest && npm run test:components && npm run test:markbind",
```

Comment:
Is there a reason why we run `markbind` after `components`, unlike `ci`, `install` and `lint` commands?

Reply:
Hmm I was going by -> unit tests -> functional tests here. Now that you mention it though, should be more consistent that way